CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -40,7 +40,8 @@ public IList<HarvestDeveloper> RetrieveDevelopers()
                     Id = (int) developer["id"],
                     FirstName = developer["first_name"].ToString(),
                     LastName = developer["last_name"].ToString(),
-                    Email = developer["email"].ToString()
+                    Email = developer["email"].ToString(),
+                    WeeklyHours = (int)developer["weekly_capacity"] / 3600
```

Comment:
This is a bit magic and could be abstracted to a private function and the `3600` could be a constant value

Reply:
Resolved this in https://github.com/madetech/crypto_tech/pull/41/commits/12fe3c4176ca837563897a91b3dd93a594462200